pkg/archlinux/create-pkgbuild.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+VERSION=$1
+
+if [ ! "${VERSION?}" ]; then
+    echo "Usage: create-pkgbuild.sh {version}"
+    exit 1
+fi
+
+cat <<- EOF
+pkgname=replay-sorcery
+pkgver=$VERSION
+pkgrel=1
+pkgdesc="An open-source, instant-replay screen recorder for Linux"
+arch=("i686" "x86_64")
+license=("GPL-3.0")
+makedepends=("cmake" "make" "git" "pkg-config" "nasm")
+url="https://github.com/matanui159/ReplaySorcery"
+source=("\${pkgname}::git+\${url}.git#tag=\${pkgver}")
+sha256sums=("SKIP")
+
+prepare() {
+  mkdir -p "\${pkgname}/build"
+  git -C "\${pkgname}" submodule update --init --recursive --depth=1
```

Comment:
Same as above although at the very least its weird that this has 2-space while above has 4-space